.circleci/config.yml

```diff
@@ -253,49 +223,20 @@ jobs:
 
       - run: *upload-pypi
 
-  build-osx-3.8:
-    <<: *osx-build-template
-    environment:
-      PYTHON: 3.8.6
-      HOMEBREW_NO_AUTO_UPDATE: 1
-      MACOSX_DEPLOYMENT_TARGET: 10.9
-
-  build-osx-3.7:
-    <<: *osx-build-template
-    environment:
-      PYTHON: 3.7.9
-      HOMEBREW_NO_AUTO_UPDATE: 1
-      MACOSX_DEPLOYMENT_TARGET: 10.9
-
-  build-osx-3.6:
-    <<: *osx-build-template
-    environment:
-      PYTHON: 3.6.12
-      HOMEBREW_NO_AUTO_UPDATE: 1
-      MACOSX_DEPLOYMENT_TARGET: 10.9
-
-  build-osx-3.5:
-    <<: *osx-build-template
-    environment:
-      PYTHON: 3.5.10
-      HOMEBREW_NO_AUTO_UPDATE: 1
-      MACOSX_DEPLOYMENT_TARGET: 10.9
-
-
 workflows:
-  version: 2
   test:
     jobs:
-      - test-3.9
-      - test-3.8
-      - test-3.7
-      - test-3.6
-      - test-3.5
-      - test-osx-3.9
-      - test-osx-3.8
-      - test-osx-3.7
-      - test-osx-3.6
-      - test-osx-3.5
+      - test-linux:
+          name: test-linux-<< matrix.python-version >>-dimod<< matrix.dimod-version >>
+          matrix:
+            parameters:
+              python-version: &python-versions [3.5.10, 3.6.8, 3.7.9, 3.8.9, 3.9.4]
+              dimod-version: [==0.9.11, <0.10.0, <0.11]
```

Comment:
Why use this set of versions? 0.9.x is tested twice. Shouldn't we add 0.8.1 as well?

Reply:
That’s true. I was following dwave-system, but the 0.9.11 test probably isn’t needed here. Will change this to `[<0.9, <0.10, <0.11]`

Reply:
Should be `['<0.9,>=0.8.1', ...` or `['~=0.8.0,>=0.8.1', ...`.